examples/elasticsearch/tilenol.yml

```diff
@@ -10,10 +10,10 @@ layers:
     minzoom: 14
     source:
       elasticsearch:
-        host: localhost
+        host: elasticsearch
         port: 9200
         index: buildings
         geometryField: geometry
         sourceFields:
           area_sqft: building.area_sqft
```

Comment:
is `area_sqft` is used for anything? Not sure if it should be removed in yaml file. i don't see it in `data.ndjson` or any reference in the go code

Reply:
Apologies, it looks like this was left in there from a copied configuration I had used as a template.